### PR TITLE
Upgrade webpacker to shakapacker

### PR DESCRIPTION
**Pull request by prasanthchaduvula**
### What
- We upgraded to v6.0.0 from webpacker v6.0.0.rc.6
- Followed steps https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md#update-steps-to-v600-from-v600rc6

### CODE REVIEW

File: Procfile.dev [hunk 1]
```diff
@@ -1,3 +1,3 @@
 web: bin/rails s -b 0.0.0.0 -p 3000
-webpacker: bin/webpack-dev-server
+webpacker: bin/webpacker-dev-server
```
Comment (prasanthchaduvula) [inline Procfile.dev:2]: Renamed the file to `bin/webpacker-dev-server`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= stylesheet_pack_tag "application" %>
-    <%= javascript_pack_tag "application"%>
+    <%= javascript_pack_tag "application" %>
```
Comment (prasanthchaduvula) [inline app/views/layouts/application.html.erb:9]: These are just prettier changes

File: bin/webpacker [hunk 3]
```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
```
Comment (prasanthchaduvula) [inline bin/webpacker:2]: Renamed the `bin/webpack` to `bin/webpacker` as discussed in the upgrading steps
https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md#update-steps-to-v600-from-v600rc6

File: bin/webpacker-dev-server [hunk 4]
```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
```
Comment (prasanthchaduvula) [inline bin/webpacker-dev-server:2]: Renamed the `bin/webpack-dev-server` to `bin/webpacker-dev-server` as discussed in the upgrading steps
https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md#update-steps-to-v600-from-v600rc6

File: config/webpack/base.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 const webpack = require("webpack");
 const aliasConfig = require("./alias");
-const { webpackConfig, merge } = require("@rails/webpacker");
+const { webpackConfig, merge } = require("shakapacker");
```
Comment (prasanthchaduvula) [inline config/webpack/base.js:3]: No more using `@rails/webpacker`

File: config/webpack/webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+ // name this file config/webpack/webpack.config.js
+ const { env, webpackConfig } = require('shakapacker')
```
Comment (prasanthchaduvula) [inline config/webpack/webpack.config.js:2]: webpack config for a single config file, config/webpack/webpack.config.j
but we will instead use the configuration file corresponding to your RAILS_ENV. If your RAILS_ENV is test, you'd be running config/webpack/test.js.

File: config/webpacker.yml [hunk 7]
```diff
@@ -45,10 +45,10 @@ development:
     allowed_hosts: "all"
     pretty: true
     headers:
-      'Access-Control-Allow-Origin': '*'
+      "Access-Control-Allow-Origin": "*"
```
Comment (prasanthchaduvula) [inline config/webpacker.yml:48]: These are just prettier changes

File: docker-compose.test.yml [hunk 8]
```diff
@@ -88,8 +88,8 @@ services:
     environment:
       discovery.type: single-node
       network.host: 0.0.0.0
-      xpack.security.enabled: 'true'
-      ELASTIC_PASSWORD: 'x2Sy5FK51zu'
+      xpack.security.enabled: "true"
```
Comment (prasanthchaduvula) [inline docker-compose.test.yml:91]: These are just prettier changes

File: docker-compose.yml [hunk 9]
```diff
@@ -88,8 +88,8 @@ services:
     environment:
       discovery.type: single-node
       network.host: 0.0.0.0
-      xpack.security.enabled: 'true'
-      ELASTIC_PASSWORD: 'x2Sy5FK51zu'
+      xpack.security.enabled: "true"
```
Comment (prasanthchaduvula) [inline docker-compose.yml:91]: These are just prettier changes

File: package.json [hunk 10]
```diff
@@ -129,7 +131,7 @@
   },
   "babel": {
     "presets": [
-      "./node_modules/@rails/webpacker/package/babel/preset.js",
+      "./node_modules/shakapacker/package/babel/preset.js",
```
Comment (prasanthchaduvula) [inline package.json:134]: No more using `@rails/webpacker`

**Comment by apoorv1316** [comment]
@prasanthchaduvula Working fine on local. Please merge when ready

**Review by vipulnsward — APPROVED** [review]

**Pull request merged**